build.rs

```diff
@@ -0,0 +1,10 @@
+winrt_macros::build!(
+    foundation
+    dependencies
+        nuget: Microsoft.Windows.SDK.Contracts
+    types
```

Comment:
observation: the syntax we have for including everything in the dependencies is a bit awkward...

Reply:
Any suggestions? I was hoping not to have to add **_any_** syntax but I couldn't figure out how to sniff out the winrt crate build without confusing all of the tests.